.prettierignore

```diff
@@ -0,0 +1,8 @@
+# Ignore artifacts:
```

Comment:
ignore nested node_modules, dist, .dcoder and .upm directories as well

Reply:
No need, tested with nested codes in dist and node_modules in the blocks, the prettierignore rules are working on nested directories too.